content/en/docs/reference/using-api/health-checks.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Kubernetes API health endpoints
+reviewers:
+- logicalhan
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+This page describes the usage of the Kubernetes API endpoints for liveness and readiness.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## API Endpoints for health
+
+Currently the Kubernetes API server supports 3 API endpoints (`healthz`, `livez`, `readyz`) to indicate the current status of the API server.
```

Comment:
nit (feel free to merge as-is):
```suggestion
The Kubernetes API server provides 3 API endpoints (`healthz`, `livez`, and `readyz`) to indicate the current status of the API server.
```
Usually I avoid “currently”; it's either not needed, or it's part of a statement about future changes (which we [avoid](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) making). The other changes are also tweaks and if you prefer your original working that's absolutely fine.